style.css

```diff
@@ -152,21 +156,203 @@ button:focus {
   width: 218px;
 }
 
-.card-back:hover {
-  transform: translate3d(-10px, -30px, -40px) rotate(-3.08deg);
-  box-shadow: 10px 10px 15px rgba(0, 0, 0, .5);
-  transition: .8s;
-  }
-
 .card-back,
 .card-front {
   position: absolute;
   -webkit-backface-visibility: hidden;
   backface-visibility: hidden;
 }
 
+.card-back:hover {
+  transform: translate3d(-10px, -30px, -40px) rotate(-3.08deg);
+  box-shadow: 10px 10px 15px rgba(0, 0, 0, .5);
+  transition: .8s;
+}
+
 .card-front {
   transform: rotateY(180deg);
 }
 
+@media (max-width: 1200px) {
+  .table-card__wrap {
+    padding-right: 0;
+    padding-bottom: 0;
+    width: 160px;
+    height: 240px;
+  }
+
+  .card-back {
+    width: 85%;
+  }
+
+  .card-front {
+    width: 85%;
+  }
+
+  .card-loser {
+    width: 100%;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/jsFinalTask/pullRequest?prid=6141138)